projects/vdk-plugins/vdk-oracle/tests/test_plugin.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+from unittest import mock
+
+import pytest
+from click.testing import Result
+from vdk.plugin.oracle import oracle_plugin
+from vdk.plugin.test_utils.util_funcs import cli_assert_equal
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
+
+
+# @pytest.fixture(scope="session", autouse=True)
+# def configure_oracle_environment():
+#     with mock.patch.dict(
+#         os.environ,
+#         {
+#             "DB_DEFAULT_TYPE": "oracle",
+#             "ORACLE_USER": "admin",
+#             "ORACLE_PASSWORD": "Gr0mh3llscr3am",
+#             "ORACLE_CONNECTION_STRING": "(description= (retry_count=20)(retry_delay=3)(address=(protocol=tcps)(port=1522)(host=adb.eu-frankfurt-1.oraclecloud.com))(connect_data=(service_name=ge975b87ba26804_ndb_high.adb.oraclecloud.com))(security=(ssl_server_dn_match=yes)))",
+#             "VDK_LOG_EXECUTION_RESULT": "True",
+#             "VDK_INGEST_METHOD_DEFAULT": "ORACLE",
+#         },
+#     ) as _fixture:
+#         yield _fixture
+#
+#
+
+
+def test_do_nothing():
+    assert True
+
+
+# def test_oracle_connect_execute():
```

Comment:
Explain why are they commented out in a comment. 

You may consider conditional execution `@pytest.mark.skipif`

Reply:
I ended up enabling the tests, they're not commented anymore.